packages/calypso-products/src/plans-list.js

```diff
@@ -650,7 +651,7 @@ export const PLANS_LIST = {
 		...getPlanBloggerDetails(),
 		term: constants.TERM_ANNUALLY,
 		getBillingTimeFrame: WPComGetBillingTimeframe,
-		availableFor: ( plan ) => includes( [ constants.PLAN_FREE ], plan ),
+		availableFor: ( plan ) => [ constants.PLAN_FREE ].includes( plan ),
```

Comment:
Feels weird to use `includes` for an array with a single element, but I guess we can leave it as-is because it wasn't introduced in this PR. Also that way it's consistent with the rest of the `availableFor` definitions.

Reply:
Yeah, I thought of the same thing but I figured that I'd leave the existing behavior.